src/helpers/esbuildbabel.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * ISC License
+ *
+ * Copyright (c) Antoine Boulanger (https://github.com/antoineboulanger)
```

Comment:
@sandeep-vedam why does it have an ISC license and is it licensed to Antoine Boulanger? Isn't this part of our code base? Then it should have a Metrological 2021 Apache 2 license, right?

Reply:
Scratch my previous comment. Just read the comment of @mhughesacn.

Reply:
I can answer this: This is because in the Blackduck tool, the code is matching to code in esbuild-plugin-babel.  The match is not perfect but with identical layout and variable names in places, it does not look original.  Therefore it needs to be attributed which means credit to the original etc. as done in RDK. - Martin (RDK CMF Compliance Team)